content/en/_index.html

```diff
@@ -24,12 +24,15 @@ <h1>The Interlisp Revival</h1>
 <p>Medley's life as a commercial product ended in the 1990s, and until 2009 its source code only remained in the hands of the development team. Now the system is running again on modern computer systems, thanks to the combined efforts of original developers and interested newcomers. Our goal is to revive Interlisp in language, environment and spirit through the Medley Interlisp Project.</p>
 
 
+<div class="col">
```

Comment:
This gets the headers and paragraphs to behave correctly.  There's probably a lot more we want to do with formatting but for now this makes it look a little better.